Tmain/interactive-mode.d/stdout-expected.txt

```diff
@@ -0,0 +1,46 @@
+identification message on startup
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+
+error on invalid command
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+{"error": "unknown command name", "fatal": true}
+
+error on missing arguments
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+{"error": "invalid generate-tags request", "fatal": true}
```

Comment:
Perhaps in error cases, we should still output a `completed` object? Like `{"completed":"invalid"}`. Might make it simpler to integrate, if you can always expect a completed object before you know to send the next command.